### PR TITLE
DEVADV-227 Update full text searching with sdk 3 docs

### PR DESCRIPTION
**Pull request by httpJunkie**
I have made changes to text, grammar, extended code examples with instructions on how to create the FTS index for the main example.

### CODE REVIEW

**Review by brett19 — APPROVED** [review]

**Comment by httpJunkie** [comment]
I am updating this PR in order to get rid of the instructions with images for setting up an index for the code examples. Instead, I will just update code examples and text/grammar. I am taking the FTS course on Couchbase first before I finish working on the code examples, I want to be able to update them so that anyone working with the travel-sample dataset can understand the examples better and aside from having the index created, the examples should be copy-paste and should be tested to work on that dataset.

This PR is not ready to merge yet.

**Comment by httpJunkie** [comment]
I have removed the instructions for creating the `index-hotel-description` and moved it into a partial file until we have a better solution of how to use it. I am creating a ticket to handle this after Richard meets with us on Wednesday of next week. If needed I can remove it altogether. Just LMK! The final doc for review is here: [full-text-searching-with-sdk](https://github.com/couchbase/docs-sdk-nodejs/blob/ix/update_full_text_searching_with_sdk_3/modules/howtos/pages/full-text-searching-with-sdk.adoc)

I have built it locally and ensured everything is working.

File: modules/howtos/pages/full-text-searching-with-sdk.adoc [hunk 1]
```diff
@@ -2,81 +2,129 @@
...
 
+The Couchbase Node.js SDK exposes an API for performing FTS queries which abstract some of the complexity of using the underlying REST API.
 
 // As of Couchbase Server 6.5, FTS...
 
 == Examples
 
-Search queries are executed at the cluster level (not bucket or collection).
-Here is a simple MatchQuery that looks for the text “swanky” using a defined index:
+Search queries are executed at the cluster level (not bucket or collection). All of the examples below will log to the console our returned documents along with their metadata and rows, each returned document has an index, id, score and sort value.
```
Comment (blbryant) [inline modules/howtos/pages/full-text-searching-with-sdk.adoc]: I assume this is against `travel-sample`?  We might want to say so.  We also might want to reference the 'getting started page for connection semantics, etc.

**Review by RichardSmedley — APPROVED** [review]
:+1:

**Review by RichardSmedley — APPROVED** [review]
:+1:

**Pull request merged**